src/routes/LandingPage/EventsBanner/EventsBanner.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+
+import React from 'react';
+import './index.scss';
+import classnames from 'classnames'
+
+const EventsBanner = () => {
+  return (
+    <div className='evts-banner'>
+      <EventsDivider highlight='1' />
+      <EventsDescription text='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.'/>
+      <EventsDivider highlight='6' />
+    </div>
+  );
+};
+//
+
+const EventsDescription = ({ text }) => {
+  return (
+    <div className='evts-desc'>
+      <p1>
+        {text}
+      </p1>
+    </div>
+  )
+}
+
+const EventsDivider = ({ highlight, rows }) => {
+  var words = [];
```

Comment:
When EventsDivider is not used in `EventsBanner`, `rows` is not passed in.
In this case, `rows` defaults to `undefined` and then we do a comparison of it with `i` which starts as `0`.
`0 < undefined` defaults to falsey i believe so it won't crash but, it may be wise to add in a check for `rows` not being undefined

Reply:
Oh, my bad, I saw the props but totally missed the default values.